src/grid.tsx

```diff
@@ -18,11 +18,11 @@ class Grid extends Component<Partial<UserConfig>, any> {
   }
 
   componentDidMount() {
-    this.instance.render(this.wrapper.current);
+    if (this.wrapper.current) this.instance.render(this.wrapper.current);
   }
 
   componentDidUpdate() {
-    this.instance.updateConfig(this.props).forceRender();
+    if (this.wrapper.current) this.instance.updateConfig(this.props).forceRender();
```

Comment:
I don't think this check is necessary here because Grid.js wrapper has already been mounted in `componentDidMount`

Reply:
Thanks @afshinm I ran into this issue on my React app. The root cause is the `_` from `gridjs-react`. It would give an error if you have a custom element on the Grid component. I already figured out the fix like this https://gist.github.com/jellydn/dc8253f95bd09fd6790125cae865425e